app/controllers/Checkout.scala

```diff
@@ -46,18 +47,30 @@ object Checkout extends Controller with LazyLogging {
           yield BadRequest(views.html.checkout.payment(formWithErrors, userIsSignedIn = idUserOpt.isDefined))
       },
       userData => {
-        CheckoutService.processSubscription(userData, AuthenticationService.authenticatedUserFor(request)).map {
-          case Some(memberId) =>
-            Redirect(routes.Checkout.thankyou())
-          case None =>
-            Redirect(routes.Checkout.renderCheckout())
-        }
+        val idUserOpt = AuthenticationService.authenticatedUserFor(request)
+        CheckoutService.processSubscription(userData, idUserOpt).map {
+          case CheckoutResult(_, guestUser: GuestUser) =>
+            Some(FinishAccountForm().bind(guestUser.toFormParams))
+          case CheckoutResult(_, _:MinimalIdUser) =>
+            None
+        }.map(formOpt => Ok(views.html.checkout.thankyou(formOpt)))
       }
     )
   }
 
   def thankyou = GoogleAuthenticatedStaffAction(Ok(views.html.checkout.thankyou()))
```

Comment:
We should still have it as it exists as a route and a user can book mark it. We need to update the page to include order subscription number ( @rtyley can you confirm?) and so we'll have to make a call to Zuora and / or Salesforce to get the data. 
I think `handleCheckout` should redirect to this instead?